Automattic/jetpack
# Instant Search: improve search phrase highlighting

Fixes https://github.com/Automattic/jetpack/issues/13792.

#### Changes proposed in this Pull Request:

We recently switched to use `<mark>` to show where search results contain the search phrase. It looks like this by default:

<img width="822" alt="Screen Shot 2019-10-22 at 16 30 41" src="https://user-images.githubusercontent.com/17325/67257658-88220900-f4e9-11e9-9fd8-103da5c74b97.png">

This PR changes the highlighting to have a more subtle appearance:

<img width="808" alt="Screen Shot 2019-10-22 at 16 27 23" src="https://user-images.githubusercontent.com/17325/67257682-9d973300-f4e9-11e9-8843-bd8fe21fb43c.png">

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
No - it's a visual adjustment to an unreleased feature.

#### Testing instructions
Add define( "JETPACK_SEARCH_PROTOTYPE", true ); to your wp-config.php.
If using Jetpack's Docker development environment, you can create a file at /docker/mu-plugins/instant-search.php and add the define there.

Ensure that your site has the Jetpack Pro plan and Jetpack Search enabled.
You can enable Jetpack Search in the Performance tab within the Jetpack menu (/wp-admin/admin.php?page=jetpack#/performance).

Perform a search and make sure the correct highlighting is applied. For example: /?s=card&blog_id=84860689 should show the word "card" bolded in results.

#### Proposed changelog entry for your changes:

* No changelog required. Merges into `instant-search-master` branch.
